package.json

```diff
@@ -25,7 +25,12 @@
     "update": "ng update @nrwl/workspace",
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
-    "help": "nx help"
+    "help": "nx help",
+    "build:schematics": "tsc -p tsconfig.schematics.json",
+    "copy:schemas": "copyfiles  schematics/*/schema.json dist/libs/angular-routing/",
+    "copy:collection": "copyfiles schematics/collection.json dist/libs/angular-routing/",
+    "schematics:postbuild": "npm run copy:schemas && npm run copy:collection",
+    "schematics:build": "npm run build:schematics && npm run schematics:postbuild"
```

Comment:
Instead let's define a new target in the angular.json, such as `build-package` to build the lib, and use `run-commands` builder to run `build-package`, and compile the schematics. See

https://github.com/ngrx/platform/blob/master/angular.json#L236

Reply:
This is nice.

Reply:
Removed.